fabtests/pytest/efa/test_rdm.py

```diff
@@ -23,18 +23,36 @@ def run_rdm_test(cmdline_args, executable, iteration_type, completion_type, memo
                           pytest.param("standard", marks=pytest.mark.standard)])
 def test_rdm_pingpong(cmdline_args, iteration_type, completion_type, memory_type):
     run_rdm_test(cmdline_args, "fi_rdm_pingpong", iteration_type,
-                 completion_type, memory_type)
+            completion_type, memory_type, "all")
+
+@pytest.mark.parametrize("message_size",
+                         [pytest.param("r:8000,4,9000", marks=pytest.mark.functional)])
+def test_rdm_pingpong_range(cmdline_args, completion_type, memory_type, message_size):
+    run_rdm_test(cmdline_args, "fi_rdm_pingpong", "short",
+                 completion_type, memory_type, message_size)
 
 @pytest.mark.parametrize("iteration_type",
                          [pytest.param("short", marks=pytest.mark.short),
                           pytest.param("standard", marks=pytest.mark.standard)])
 def test_rdm_tagged_pingpong(cmdline_args, iteration_type, completion_type, memory_type):
     run_rdm_test(cmdline_args, "fi_rdm_tagged_pingpong", iteration_type,
-                 completion_type, memory_type)
+                 completion_type, memory_type, "all")
+
+@pytest.mark.parametrize("message_size",
+                         [pytest.param("r:8000,4,9000", marks=pytest.mark.functional)])
```

Comment:
Can add 2 more ranges:  [0,4,64] (where we cross device's inline range), and [4048,4,4148] (around 4096, which is shm protocol cross point)

Reply:
Done